src/components/Sidebar.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <nav :class="{active}">
     <template v-if="isLogged">
-      <router-link class="action" to="/files" :aria-label="$t('sidebar.myFiles')" :title="$t('sidebar.myFiles')">
+      <router-link class="action" to="/files" :aria-label="$t('sidebar.myFiles')" :title="$t('sidebar.myFiles.title')">
         <i class="material-icons">folder</i>
-        <span>{{ $t('sidebar.myFiles') }}</span>
+        <span>{{ $t('sidebar.myFiles.personalized', {person: user.firstName}) }}</span>
```

Comment:
add condition on personalized to check if exist, if not fallback to title

Reply:
done